src/node_util.cc

```diff
@@ -262,28 +263,25 @@ static void GetCallSite(const FunctionCallbackInfo<Value>& args) {
 
   // Frame 0 is node:util. It should be skipped.
   for (int i = 1; i < frame_count; ++i) {
-    Local<Object> obj = Object::New(isolate);
     Local<StackFrame> stack_frame = stack->GetFrame(isolate, i);
 
     Utf8Value function_name(isolate, stack_frame->GetFunctionName());
     Utf8Value script_name(isolate, stack_frame->GetScriptName());
 
-    obj->Set(env->context(),
-             env->function_name_string(),
-             String::NewFromUtf8(isolate, *function_name).ToLocalChecked())
-        .Check();
-    obj->Set(env->context(),
-             env->script_name_string(),
-             String::NewFromUtf8(isolate, *script_name).ToLocalChecked())
-        .Check();
-    obj->Set(env->context(),
-             env->line_number_string(),
-             Integer::NewFromUnsigned(isolate, stack_frame->GetLineNumber()))
-        .Check();
-    obj->Set(env->context(),
-             env->column_string(),
-             Integer::NewFromUnsigned(isolate, stack_frame->GetColumn()))
-        .Check();
+    Local<Name> names[] = {
+        env->function_name_string(),
+        env->script_name_string(),
+        env->line_number_string(),
+        env->column_string(),
+    };
+    Local<Value> values[] = {
+        String::NewFromUtf8(isolate, *function_name).ToLocalChecked(),
```

Comment:
Why does this need a copy? It seems fine to simply use `stack_frame->GetFunctionName()` here and not go through the UTF8 decoding-encoding.

Reply:
I think I just copied/paste when I was using `obj->Set`. Let me try to adjust it.

Reply:
So, as I told you via Slack, I tried some approaches (see diff)

```diff
diff --git a/src/node_util.cc b/src/node_util.cc
index 0a188caf16..92d548fd83 100644
--- a/src/node_util.cc
+++ b/src/node_util.cc
@@ -250,6 +250,7 @@ static void ParseEnv(const FunctionCallbackInfo<Value>& args) {
 static void GetCallSite(const FunctionCallbackInfo<Value>& args) {
   Environment* env = Environment::GetCurrent(args);
   Isolate* isolate = env->isolate();
+  v8::HandleScope handle_scope(isolate);
 
   CHECK_EQ(args.Length(), 1);
   CHECK(args[0]->IsNumber());
@@ -275,8 +276,8 @@ static void GetCallSite(const FunctionCallbackInfo<Value>& args) {
         env->column_string(),
     };
     Local<Value> values[] = {
-        String::NewFromUtf8(isolate, *function_name).ToLocalChecked(),
-        String::NewFromUtf8(isolate, *script_name).ToLocalChecked(),
+        stack_frame->GetFunctionName(),
+        stack_frame->GetScriptName(),
         Integer::NewFromUnsigned(isolate, stack_frame->GetLineNumber()),
         Integer::NewFromUnsigned(isolate, stack_frame->GetColumn()),
     };
```

but all of them caused segmentation fault.